src/app/main/component/events/components/images-container/images-container.component.html

```diff
@@ -2,15 +2,15 @@
   <div clas="image-container">
     <div class="row">
       <div class="title-wrapper">
-        <mat-label>Picture</mat-label>
-        <mat-label>{{ imageCount }}/5</mat-label>
+        <mat-label>{{ 'create-event.picture' | translate }}</mat-label>
+        <mat-label>{{ imageCount }}{{ 'create-event.picture-length' | translate }}</mat-label>
```

Comment:
The same, replace {{ 'create-event.picture-length' | translate }} with /5

Reply:
Corrected the errors as you recommended

Reply:
Forgot to delete translation for X